lib/mayaUsd/ufe/UsdTRSUndoableCommandBase.cpp

```diff
@@ -17,41 +17,447 @@
 #include "UsdTRSUndoableCommandBase.h"
 #include "private/Utils.h"
 
+#include <pxr/base/gf/rotation.h>
+
 #include <ufe/scene.h>
 #include <ufe/sceneNotification.h>
 
+#include <iostream>
+
 MAYAUSD_NS_DEF {
 namespace ufe {
 
-template<class V>
-UsdTRSUndoableCommandBase<V>::UsdTRSUndoableCommandBase(
-    const UsdSceneItem::Ptr& item, double x, double y, double z
-) : fItem(item), fNewValue(x, y, z)
-{}
+namespace {
+
+// Placeholder for delivering an op-suffix via env-var or Maya setting
+std::string getOpSuffix()
+{
+    // os.environ['UFE_XFORMOP_SUFFIX']='??' to use a new suffix
+    const char* UFE_XFORMOP_SUFFIX = ::getenv("UFE_XFORM_OP_SUFFIX");
+
+    return UFE_XFORMOP_SUFFIX ? UFE_XFORMOP_SUFFIX : "";
```

Comment:
This won't work. We need to support all of the naming conventions, not just one.